packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -7,6 +7,102 @@ section: get-started
 import './get-started.css';
 import { Divider, Timestamp } from '@patternfly/react-core';
 
+<Timestamp>March 2024</Timestamp>
+
+## PatternFly 6.2
+
+### Promoted package versions
+- patternfly/patternfly ([changelog](https://github.com/patternfly/patternfly/releases/tag/v6.2.0))
+    - [@patternfly/patternfly@6.2.0](https://www.npmjs.com/package/@patternfly/patternfly)
+- patternfly/react ([changelog](https://github.com/patternfly/patternfly-react/releases/tag/v6.2.0))
+- PatternFly extensions 
+    - [@patternfly/chatbot@2.2.0](https://www.npmjs.com/package/@patternfly/chatbot) ([changelog](https://github.com/patternfly/chatbot/releases/tag/v2.2.0))
+
+### Component updates
+
+#### Bugs 
+- Select: We fixed scrolling for menus with grouped items.
+- Masthead: We correct vertical centering.
+
+#### Features 
+-
+
+### Extension updates 
+
+#### ChatBot
+
+##### Bugs
+- We fixed an overflow issue in popovers with long titles.
+- We fixed the appearance of links.
+- We removed the cancel button from the user feedback form, which was unnecessary due to the presence of a close button.
+- We made resize and form-control overrides more important to assist with downstream implementations.
+- We added loading and error states to the conversation history.
+- We made updates for general style consistency.
+
```

Comment:
* We are now allowing consumers to pass additional [rehype](https://github.com/rehypejs/rehype) plugins into Messages for further customization of Markdown parsing.
* Replaced MessageBar with PatternFly TextArea. It can now also be made controllable, similar to other PatternFly inputs. This did mean a couple of type changes for MessageBarProps. onChange has changed from `(event: React.ChangeEvent<HTMLDivElement>, value: string) => void;` to ` onChange?: (event: React.ChangeEvent<HTMLTextAreaElement>, value: string | number) => void;`. onSendMessage has changed from `onSendMessage: (message: string) => void;` to `(message: string | number) => void;`